crates/red_knot_python_semantic/src/types.rs

```diff
@@ -367,6 +371,15 @@ impl<'db> Type<'db> {
         }
     }
 
+    pub fn is_stdlib_symbol(&self, db: &'db dyn Db, module_name: &str, name: &str) -> bool {
+        match self {
+            Type::Class(class) => class.is_stdlib_symbol(db, module_name, name),
+            Type::Function(function) => function.is_stdlib_symbol(db, module_name, name),
+            Type::RevealTypeFunction(_) => true,
```

Comment:
Is this correct? It surprises me that `Type::RevealedType(...).is_stdlib_symbol(&db, "collections", "deque")` would return `true`. It's not the stdlib symbol `collections.deque`!

Reply:
Oops, no, this is wrong, good catch! I originally had it correct, and then in a review pass I was like "wait, we always know it's a stdlib symbol", forgetting that the semantics of the function are not "is it a stdlib symbol" but rather "is it _this_ stdlib symbol". Fixed.